### PR TITLE
Refactor parameters for Google symptoms

### PR DESCRIPTION
**Pull request by sgsmob**
### Description
Update `google_symptoms` `params.json` files in accordance with #795.  Also removes some unused parameters.

### CODE REVIEW

File: google_symptoms/params.json.template [hunk 1]
```diff
@@ -1,8 +1,12 @@
...
-  "path_to_bigquery_credentials": "credentials.json",
-  "log_exceptions": false
+  "common": {
+    "export_dir": "./receiving",
+    "log_exceptions": false
+  },
+  "indicator": {
+    "export_start_date": "2020-02-20",
+    "num_export_days": 14,
+    "path_to_bigquery_credentials": "credentials.json",
```
Comment (chinandrew) [inline google_symptoms/params.json.template:9]: unrelated to this PR as a refactor, but why don't these appear in the ansible json?

Comment (sgsmob) [reply google_symptoms/params.json.template:9]: @nmdefries I think you may have added this line initially

Comment (nmdefries) [reply google_symptoms/params.json.template:9]: Yes, these params should also appear in the Ansible json. @sgsmob you can add them if you want or I'll make a followup PR.

Comment (sgsmob) [reply google_symptoms/params.json.template:9]: I will add it here, thanks for the info

Comment (sgsmob) [reply google_symptoms/params.json.template:9]: Done

**Review by chinandrew — APPROVED** [review]
:shipit::shipit::shipit:

**Pull request merged**